### ISSUE TITLE
Not able to launch Firefox version 53 and 54 .

### ISSUE THREAD

**Issue by punitbisht** [arquillian/arquillian-extension-drone#100]
##### Issue Overview

Tests are not able to launch firefox version 53 and version 54, same test work fine in firefox 52.0 

##### Expected Behaviour

should launch firefox 54 

##### Current Behaviour
Tests are not running on firefox 54 .

##### Additional Information

Anything relevant to help us resolving the problem. For example if you are using `mvn` you can share output for `mvn --version` as well as dependencies in use `mvn dependency:tree`. OS as well as JDK version would be helpful too.

For long outputs such as stacktraces please use HTML5 `<details>`

##### system details 

Mac OSX 10.12.2

##### pom.xml details 
```
<dependencyManagement>

        <dependencies>

            <!-- Arquillian Core dependencies -->
            <dependency>
                <groupId>org.jboss.arquillian</groupId>
                <artifactId>arquillian-bom</artifactId>
                <version>1.1.13.Final</version>
                <type>pom</type>
                <scope>import</scope>
                <exclusions><exclusion>
                    <groupId>com.google.guava</groupId>
                    <artifactId>guava</artifactId>
                </exclusion></exclusions>

            </dependency>
            <!-- Arquillian Drone dependencies and WebDriver/Selenium dependencies -->
            <dependency>
                <groupId>org.jboss.arquillian.extension</groupId>
                <artifactId>arquillian-drone-bom</artifactId>
                <version>2.2.0</version>
                <type>pom</type>
                <scope>import</scope>
                <exclusions><exclusion>
                    <groupId>com.google.guava</groupId>
                    <artifactId>guava</artifactId>
                </exclusion></exclusions>
            </dependency>
            <dependency>
                <groupId>org.arquillian.rusheye</groupId>
                <artifactId>rusheye-impl</artifactId>
                <version>1.0.0</version>
            </dependency>
        </dependencies>

    </dependencyManagement>
    <dependencies>

        <dependency>
            <groupId>com.google.guava</groupId>
            <artifactId>guava</artifactId>
            <version>21.0</version>
        </dependency>
        <dependency>
            <groupId>org.jboss.arquillian.graphene</groupId>
            <artifactId>graphene-webdriver</artifactId>
            <version>2.2.1</version>
            <type>pom</type>
            <scope>test</scope>
        </dependency>

        <dependency>
            <groupId>org.jboss.arquillian.junit</groupId>
            <artifactId>arquillian-junit-standalone</artifactId>
            <version>${arquillian.version}</version>
            <scope>test</scope>

        </dependency>
.
.
.
.
  </dependencies>
```
##### debug logs 
Jun 29, 2017 9:08:47 AM org.jboss.arquillian.drone.webdriver.factory.remote.reusable.ReusedSessionPermanentFileStorage readStore
INFO: Reused session store is not available at /Users/ubishpu/.drone-webdriver-session-store, a new one will be created.
Jun 29, 2017 9:08:47 AM org.jboss.arquillian.drone.impl.DroneTestEnricher ensureInjectionPointPrepared
WARNING: Injection point DronePointImpl{droneClass=interface org.openqa.selenium.WebDriver, annotations=[@org.jboss.arquillian.drone.api.annotation.Drone()], lifecycle=CLASS} was not prepared yet. It will be prepared now, but it's recommended that all drones with class lifecycle are prepared in @BeforeClass by the DroneLifecycleManager!
1498745329181	geckodriver	INFO	Listening on 127.0.0.1:26587
1498745329434	geckodriver::marionette	INFO	Starting browser /Applications/Firefox.app/Contents/MacOS/firefox-bin with args ["-marionette"]
1498745329667	addons.manager	DEBUG	Application has been upgraded
1498745329693	addons.manager	DEBUG	Loaded provider scope for resource://gre/modules/addons/XPIProvider.jsm: ["XPIProvider"]
1498745329695	addons.manager	DEBUG	Loaded provider scope for resource://gre/modules/LightweightThemeManager.jsm: ["LightweightThemeManager"]
1498745329698	addons.manager	DEBUG	Loaded provider scope for resource://gre/modules/addons/GMPProvider.jsm
1498745329701	addons.manager	DEBUG	Loaded provider scope for resource://gre/modules/addons/PluginProvider.jsm
1498745329702	addons.manager	DEBUG	Starting provider: XPIProvider
1498745329702	addons.xpi	DEBUG	startup
1498745329703	addons.xpi	INFO	Removing all system add-on upgrades.
1498745329703	addons.xpi	INFO	SystemAddonInstallLocation directory is missing
1498745329704	addons.xpi	INFO	Mapping aushelper@mozilla.org to /Applications/Firefox.app/Contents/Resources/browser/features/aushelper@mozilla.org.xpi
1498745329704	addons.xpi	INFO	Mapping e10srollout@mozilla.org to /Applications/Firefox.app/Contents/Resources/browser/features/e10srollout@mozilla.org.xpi
1498745329705	addons.xpi	INFO	Mapping firefox@getpocket.com to /Applications/Firefox.app/Contents/Resources/browser/features/firefox@getpocket.com.xpi
1498745329705	addons.xpi	INFO	Mapping screenshots@mozilla.org to /Applications/Firefox.app/Contents/Resources/browser/features/screenshots@mozilla.org.xpi
1498745329705	addons.xpi	INFO	Mapping webcompat@mozilla.org to /Applications/Firefox.app/Contents/Resources/browser/features/webcompat@mozilla.org.xpi
1498745329705	addons.xpi	INFO	Mapping {972ce4c6-7e08-4474-a285-3208198ce6fd} to /Applications/Firefox.app/Contents/Resources/browser/extensions/{972ce4c6-7e08-4474-a285-3208198ce6fd}.xpi
1498745329706	addons.xpi	DEBUG	checkForChanges
1498745329706	addons.xpi	INFO	SystemAddonInstallLocation directory is missing
1498745329706	addons.xpi	DEBUG	Loaded add-on state from prefs: {}
1498745329707	addons.xpi	DEBUG	New add-on aushelper@mozilla.org in app-system-defaults
1498745329708	addons.xpi	DEBUG	getModTime: Recursive scan of aushelper@mozilla.org
1498745329708	addons.xpi	DEBUG	New add-on e10srollout@mozilla.org in app-system-defaults
1498745329709	addons.xpi	DEBUG	getModTime: Recursive scan of e10srollout@mozilla.org
1498745329709	addons.xpi	DEBUG	New add-on firefox@getpocket.com in app-system-defaults
1498745329710	addons.xpi	DEBUG	getModTime: Recursive scan of firefox@getpocket.com
1498745329710	addons.xpi	DEBUG	New add-on screenshots@mozilla.org in app-system-defaults
1498745329710	addons.xpi	DEBUG	getModTime: Recursive scan of screenshots@mozilla.org
1498745329710	addons.xpi	DEBUG	New add-on webcompat@mozilla.org in app-system-defaults
1498745329711	addons.xpi	DEBUG	getModTime: Recursive scan of webcompat@mozilla.org
1498745329711	addons.xpi	DEBUG	New add-on {972ce4c6-7e08-4474-a285-3208198ce6fd} in app-global
1498745329711	addons.xpi	DEBUG	getModTime: Recursive scan of {972ce4c6-7e08-4474-a285-3208198ce6fd}
1498745329711	addons.xpi	DEBUG	getInstallState changed: true, state: {"app-system-defaults":{"aushelper@mozilla.org":{"d":"/Applications/Firefox.app/Contents/Resources/browser/features/aushelper@mozilla.org.xpi","st":1496948382000},"e10srollout@mozilla.org":{"d":"/Applications/Firefox.app/Contents/Resources/browser/features/e10srollout@mozilla.org.xpi","st":1496948382000},"firefox@getpocket.com":{"d":"/Applications/Firefox.app/Contents/Resources/browser/features/firefox@getpocket.com.xpi","st":1496948382000},"screenshots@mozilla.org":{"d":"/Applications/Firefox.app/Contents/Resources/browser/features/screenshots@mozilla.org.xpi","st":1496948383000},"webcompat@mozilla.org":{"d":"/Applications/Firefox.app/Contents/Resources/browser/features/webcompat@mozilla.org.xpi","st":1496948383000}},"app-global":{"{972ce4c6-7e08-4474-a285-3208198ce6fd}":{"d":"/Applications/Firefox.app/Contents/Resources/browser/extensions/{972ce4c6-7e08-4474-a285-3208198ce6fd}.xpi","st":1496948382000}}}
1498745329719	addons.xpi-utils	DEBUG	Opening XPI database /var/folders/0v/9tqxsqpn38n64cwv8b6mdh9w3d7_zq/T/rust_mozprofile.Vpw2ILYUh1wM/extensions.json
1498745329720	addons.xpi-utils	DEBUG	New add-on aushelper@mozilla.org installed in app-system-defaults
*** Blocklist::_loadBlocklistFromFile: blocklist is disabled
1498745329728	DeferredSave.extensions.json	DEBUG	Save changes
1498745329728	addons.xpi-utils	DEBUG	New add-on e10srollout@mozilla.org installed in app-system-defaults
1498745329731	DeferredSave.extensions.json	DEBUG	Starting timer
1498745329735	DeferredSave.extensions.json	DEBUG	Save changes
1498745329735	addons.xpi-utils	DEBUG	New add-on firefox@getpocket.com installed in app-system-defaults
1498745329740	DeferredSave.extensions.json	DEBUG	Save changes
1498745329740	addons.xpi-utils	DEBUG	New add-on screenshots@mozilla.org installed in app-system-defaults
1498745329745	DeferredSave.extensions.json	DEBUG	Save changes
1498745329747	addons.xpi-utils	DEBUG	New add-on webcompat@mozilla.org installed in app-system-defaults
1498745329755	DeferredSave.extensions.json	DEBUG	Save changes
1498745329755	addons.xpi-utils	DEBUG	New add-on {972ce4c6-7e08-4474-a285-3208198ce6fd} installed in app-global
1498745329759	DeferredSave.extensions.json	DEBUG	Save changes
1498745329760	addons.manager	DEBUG	Registering startup change 'installed' for aushelper@mozilla.org
1498745329767	addons.xpi	DEBUG	Loading bootstrap scope from /Applications/Firefox.app/Contents/Resources/browser/features/aushelper@mozilla.org.xpi
1498745329771	addons.xpi	DEBUG	Calling bootstrap method install on aushelper@mozilla.org version 2.0
1498745329771	addons.xpi-utils	DEBUG	Make addon app-system-defaults:aushelper@mozilla.org visible
1498745329772	DeferredSave.extensions.json	DEBUG	Save changes
1498745329772	addons.manager	DEBUG	Registering startup change 'installed' for e10srollout@mozilla.org
1498745329773	addons.xpi	DEBUG	Loading bootstrap scope from /Applications/Firefox.app/Contents/Resources/browser/features/e10srollout@mozilla.org.xpi
1498745329774	addons.xpi	DEBUG	Calling bootstrap method install on e10srollout@mozilla.org version 1.50
1498745329775	addons.xpi-utils	DEBUG	Make addon app-system-defaults:e10srollout@mozilla.org visible
1498745329776	DeferredSave.extensions.json	DEBUG	Save changes
1498745329776	addons.manager	DEBUG	Registering startup change 'installed' for firefox@getpocket.com
1498745329776	addons.xpi	DEBUG	Loading bootstrap scope from /Applications/Firefox.app/Contents/Resources/browser/features/firefox@getpocket.com.xpi
1498745329779	addons.xpi	DEBUG	Calling bootstrap method install on firefox@getpocket.com version 1.0.5
1498745329779	addons.xpi-utils	DEBUG	Make addon app-system-defaults:firefox@getpocket.com visible
1498745329780	DeferredSave.extensions.json	DEBUG	Save changes
1498745329780	addons.manager	DEBUG	Registering startup change 'installed' for screenshots@mozilla.org
1498745329780	addons.xpi	DEBUG	Loading bootstrap scope from /Applications/Firefox.app/Contents/Resources/browser/features/screenshots@mozilla.org.xpi
1498745329782	addons.xpi	DEBUG	Calling bootstrap method install on screenshots@mozilla.org version 6.6.0
1498745329782	addons.xpi-utils	DEBUG	Make addon app-system-defaults:screenshots@mozilla.org visible
1498745329783	DeferredSave.extensions.json	DEBUG	Save changes
1498745329783	addons.manager	DEBUG	Registering startup change 'installed' for webcompat@mozilla.org
1498745329784	addons.xpi	DEBUG	Loading bootstrap scope from /Applications/Firefox.app/Contents/Resources/browser/features/webcompat@mozilla.org.xpi
1498745329787	addons.xpi	DEBUG	Calling bootstrap method install on webcompat@mozilla.org version 1.1
1498745329787	addons.xpi-utils	DEBUG	Make addon app-system-defaults:webcompat@mozilla.org visible
1498745329788	DeferredSave.extensions.json	DEBUG	Save changes
1498745329788	addons.xpi-utils	DEBUG	Make addon app-global:{972ce4c6-7e08-4474-a285-3208198ce6fd} visible
1498745329789	DeferredSave.extensions.json	DEBUG	Save changes
1498745329789	addons.xpi	DEBUG	Updating XPIState for {"id":"aushelper@mozilla.org","syncGUID":"{92412f8d-ffea-a44b-b263-24cff640cfcf}","location":"app-system-defaults","version":"2.0","type":"extension","internalName":null,"updateURL":null,"updateKey":null,"optionsURL":null,"optionsType":null,"aboutURL":null,"icons":{},"iconURL":null,"icon64URL":null,"defaultLocale":{"name":"Application Update Service Helper","description":"Sets value(s) in the update url based on custom checks.","creator":null,"homepageURL":null},"visible":true,"active":true,"userDisabled":false,"appDisabled":false,"descriptor":"/Applications/Firefox.app/Contents/Resources/browser/features/aushelper@mozilla.org.xpi","installDate":1496948382000,"updateDate":1496948382000,"applyBackgroundUpdates":1,"bootstrap":true,"skinnable":false,"size":8486,"sourceURI":null,"releaseNotesURI":null,"softDisabled":false,"foreignInstall":false,"hasBinaryComponents":false,"strictCompatibility":false,"locales":[],"targetApplications":[{"id":"{ec8030f7-c20a-464f-9b0e-13a3a9e97384}","minVersion":"54.0","maxVersion":"54.*"}],"targetPlatforms":[],"multiprocessCompatible":true,"seen":true,"dependencies":[],"hasEmbeddedWebExtension":false,"mpcOptedOut":false,"userPermissions":null}
1498745329789	addons.xpi	DEBUG	getModTime: Recursive scan of aushelper@mozilla.org
1498745329790	addons.xpi	DEBUG	Updating XPIState for {"id":"e10srollout@mozilla.org","syncGUID":"{7be61400-711b-f941-8dfd-ee98bf02c1b3}","location":"app-system-defaults","version":"1.50","type":"extension","internalName":null,"updateURL":null,"updateKey":null,"optionsURL":null,"optionsType":null,"aboutURL":null,"icons":{},"iconURL":null,"icon64URL":null,"defaultLocale":{"name":"Multi-process staged rollout","description":"Staged rollout of Firefox multi-process feature.","creator":null,"homepageURL":null},"visible":true,"active":true,"userDisabled":false,"appDisabled":false,"descriptor":"/Applications/Firefox.app/Contents/Resources/browser/features/e10srollout@mozilla.org.xpi","installDate":1496948382000,"updateDate":1496948382000,"applyBackgroundUpdates":1,"bootstrap":true,"skinnable":false,"size":12517,"sourceURI":null,"releaseNotesURI":null,"softDisabled":false,"foreignInstall":false,"hasBinaryComponents":false,"strictCompatibility":false,"locales":[],"targetApplications":[{"id":"{ec8030f7-c20a-464f-9b0e-13a3a9e97384}","minVersion":"54.0","maxVersion":"54.*"}],"targetPlatforms":[],"multiprocessCompatible":true,"seen":true,"dependencies":[],"hasEmbeddedWebExtension":false,"mpcOptedOut":false,"userPermissions":null}
1498745329790	addons.xpi	DEBUG	getModTime: Recursive scan of e10srollout@mozilla.org
1498745329790	addons.xpi	DEBUG	Updating XPIState for {"id":"firefox@getpocket.com","syncGUID":"{360a5895-e5d4-c841-96ba-587e087a13f5}","location":"app-system-defaults","version":"1.0.5","type":"extension","internalName":null,"updateURL":null,"updateKey":null,"optionsURL":null,"optionsType":null,"aboutURL":null,"icons":{},"iconURL":null,"icon64URL":null,"defaultLocale":{"name":"Pocket","description":"When you find something you want to view later, put it in Pocket.","creator":null,"homepageURL":null},"visible":true,"active":true,"userDisabled":false,"appDisabled":false,"descriptor":"/Applications/Firefox.app/Contents/Resources/browser/features/firefox@getpocket.com.xpi","installDate":1496948382000,"updateDate":1496948382000,"applyBackgroundUpdates":1,"bootstrap":true,"skinnable":false,"size":913904,"sourceURI":null,"releaseNotesURI":null,"softDisabled":false,"foreignInstall":false,"hasBinaryComponents":false,"strictCompatibility":false,"locales":[],"targetApplications":[{"id":"{ec8030f7-c20a-464f-9b0e-13a3a9e97384}","minVersion":"54.0","maxVersion":"54.*"}],"targetPlatforms":[],"multiprocessCompatible":true,"seen":true,"dependencies":[],"hasEmbeddedWebExtension":false,"mpcOptedOut":false,"userPermissions":null}
1498745329790	addons.xpi	DEBUG	getModTime: Recursive scan of firefox@getpocket.com
1498745329791	addons.xpi	DEBUG	Updating XPIState for {"id":"screenshots@mozilla.org","syncGUID":"{55e6309f-4cba-7e48-b9bf-45435894e48e}","location":"app-system-defaults","version":"6.6.0","type":"extension","internalName":null,"updateURL":null,"updateKey":null,"optionsURL":null,"optionsType":null,"aboutURL":null,"icons":{},"iconURL":null,"icon64URL":null,"defaultLocale":{"name":"Firefox Screenshots","description":null,"creator":null,"homepageURL":"https://pageshot.net/"},"visible":true,"active":true,"userDisabled":false,"appDisabled":false,"descriptor":"/Applications/Firefox.app/Contents/Resources/browser/features/screenshots@mozilla.org.xpi","installDate":1496948383000,"updateDate":1496948383000,"applyBackgroundUpdates":1,"bootstrap":true,"skinnable":false,"size":716271,"sourceURI":null,"releaseNotesURI":null,"softDisabled":false,"foreignInstall":false,"hasBinaryComponents":false,"strictCompatibility":false,"locales":[],"targetApplications":[{"id":"{ec8030f7-c20a-464f-9b0e-13a3a9e97384}","minVersion":"51.0a1","maxVersion":"*"}],"targetPlatforms":[],"multiprocessCompatible":true,"seen":true,"dependencies":[],"hasEmbeddedWebExtension":false,"mpcOptedOut":false,"userPermissions":null}
1498745329791	addons.xpi	DEBUG	getModTime: Recursive scan of screenshots@mozilla.org
1498745329791	addons.xpi	DEBUG	Updating XPIState for {"id":"webcompat@mozilla.org","syncGUID":"{582e97e5-53f3-3a47-a5d7-f936686be738}","location":"app-system-defaults","version":"1.1","type":"extension","internalName":null,"updateURL":null,"updateKey":null,"optionsURL":null,"optionsType":null,"aboutURL":null,"icons":{},"iconURL":null,"icon64URL":null,"defaultLocale":{"name":"Web Compat","description":"Urgent post-release fixes for web compatibility.","creator":null,"homepageURL":null},"visible":true,"active":true,"userDisabled":false,"appDisabled":false,"descriptor":"/Applications/Firefox.app/Contents/Resources/browser/features/webcompat@mozilla.org.xpi","installDate":1496948383000,"updateDate":1496948383000,"applyBackgroundUpdates":1,"bootstrap":true,"skinnable":false,"size":9004,"sourceURI":null,"releaseNotesURI":null,"softDisabled":false,"foreignInstall":false,"hasBinaryComponents":false,"strictCompatibility":false,"locales":[],"targetApplications":[{"id":"{ec8030f7-c20a-464f-9b0e-13a3a9e97384}","minVersion":"54.0","maxVersion":"54.*"}],"targetPlatforms":[],"multiprocessCompatible":true,"seen":true,"dependencies":[],"hasEmbeddedWebExtension":false,"mpcOptedOut":false,"userPermissions":null}
1498745329791	addons.xpi	DEBUG	getModTime: Recursive scan of webcompat@mozilla.org
1498745329792	addons.xpi	DEBUG	Updating XPIState for {"id":"{972ce4c6-7e08-4474-a285-3208198ce6fd}","syncGUID":"{5e03f95e-3f77-f841-8ef5-5130330d0d1f}","location":"app-global","version":"54.0","type":"theme","internalName":"classic/1.0","updateURL":null,"updateKey":null,"optionsURL":null,"optionsType":null,"aboutURL":null,"icons":{},"iconURL":"chrome://browser/content/default-theme-icon.svg","icon64URL":null,"defaultLocale":{"name":"Default","description":"The default theme.","creator":"Mozilla","homepageURL":null,"contributors":["Mozilla Contributors"]},"visible":true,"active":true,"userDisabled":false,"appDisabled":false,"descriptor":"/Applications/Firefox.app/Contents/Resources/browser/extensions/{972ce4c6-7e08-4474-a285-3208198ce6fd}.xpi","installDate":1496948382000,"updateDate":1496948382000,"applyBackgroundUpdates":1,"skinnable":true,"size":3908,"sourceURI":null,"releaseNotesURI":null,"softDisabled":false,"foreignInstall":false,"hasBinaryComponents":false,"strictCompatibility":true,"locales":[],"targetApplications":[{"id":"{ec8030f7-c20a-464f-9b0e-13a3a9e97384}","minVersion":"54.0","maxVersion":"54.0"}],"targetPlatforms":[],"seen":true,"dependencies":[],"hasEmbeddedWebExtension":false,"userPermissions":null}
1498745329792	addons.xpi	DEBUG	getModTime: Recursive scan of {972ce4c6-7e08-4474-a285-3208198ce6fd}
1498745329792	DeferredSave.extensions.json	DEBUG	Save changes
1498745329792	addons.xpi	DEBUG	Updating database with changes to installed add-ons
1498745329792	addons.xpi-utils	DEBUG	Updating add-on states
1498745329793	addons.xpi-utils	DEBUG	Writing add-ons list
1498745329794	addons.xpi	DEBUG	Registering manifest for /Applications/Firefox.app/Contents/Resources/browser/features/aushelper@mozilla.org.xpi
1498745329795	addons.xpi	DEBUG	Calling bootstrap method startup on aushelper@mozilla.org version 2.0
1498745329795	addons.xpi	DEBUG	Registering manifest for /Applications/Firefox.app/Contents/Resources/browser/features/e10srollout@mozilla.org.xpi
1498745329796	addons.xpi	DEBUG	Calling bootstrap method startup on e10srollout@mozilla.org version 1.50
1498745329796	addons.xpi	DEBUG	Registering manifest for /Applications/Firefox.app/Contents/Resources/browser/features/firefox@getpocket.com.xpi
1498745329797	addons.xpi	DEBUG	Calling bootstrap method startup on firefox@getpocket.com version 1.0.5
1498745329797	addons.xpi	DEBUG	Registering manifest for /Applications/Firefox.app/Contents/Resources/browser/features/screenshots@mozilla.org.xpi
1498745329798	addons.xpi	DEBUG	Calling bootstrap method startup on screenshots@mozilla.org version 6.6.0
1498745329798	addons.xpi	DEBUG	Registering manifest for /Applications/Firefox.app/Contents/Resources/browser/features/webcompat@mozilla.org.xpi
1498745329798	addons.xpi	DEBUG	Calling bootstrap method startup on webcompat@mozilla.org version 1.1
1498745329803	addons.manager	DEBUG	Registering shutdown blocker for XPIProvider
1498745329804	addons.manager	DEBUG	Provider finished startup: XPIProvider
1498745329804	addons.manager	DEBUG	Starting provider: LightweightThemeManager
1498745329804	addons.manager	DEBUG	Registering shutdown blocker for LightweightThemeManager
1498745329805	addons.manager	DEBUG	Provider finished startup: LightweightThemeManager
1498745329805	addons.manager	DEBUG	Starting provider: GMPProvider
1498745329809	addons.manager	DEBUG	Registering shutdown blocker for GMPProvider
1498745329809	addons.manager	DEBUG	Provider finished startup: GMPProvider
1498745329809	addons.manager	DEBUG	Starting provider: PluginProvider
1498745329809	addons.manager	DEBUG	Registering shutdown blocker for PluginProvider
1498745329809	addons.manager	DEBUG	Provider finished startup: PluginProvider
1498745329810	addons.manager	DEBUG	Completed startup sequence
1498745330125	addons.manager	DEBUG	Starting provider: <unnamed-provider>
1498745330125	addons.manager	DEBUG	Registering shutdown blocker for <unnamed-provider>
1498745330125	addons.manager	DEBUG	Provider finished startup: <unnamed-provider>
1498745330146	addons.manager	DEBUG	Starting provider: PreviousExperimentProvider
1498745330146	addons.manager	DEBUG	Registering shutdown blocker for PreviousExperimentProvider
1498745330146	addons.manager	DEBUG	Provider finished startup: PreviousExperimentProvider
1498745330152	DeferredSave.extensions.json	DEBUG	Starting write
1498745330269	DeferredSave.extensions.json	DEBUG	Write succeeded
1498745330270	addons.xpi-utils	DEBUG	XPI Database saved, setting schema version preference to 19
1498745331320	Marionette	INFO	Listening on port 53284
1498745331402	Marionette	WARN	TLS certificate errors will be ignored for this session
Jun 29, 2017 9:08:51 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: W3C
1498745391169	addons.productaddons	INFO	sending request to: https://aus5.mozilla.org/update/3/GMP/54.0/20170608105825/Darwin_x86_64-gcc3/en-US/release/Darwin%2016.3.0/default/default/update.xml
1498745391600	addons.productaddons	INFO	Completed downloading document
1498745392051	addons.productaddons	INFO	downloadXHR File download. status=200
1498745392054	addons.productaddons	INFO	Downloaded file will be saved to /private/var/folders/0v/9tqxsqpn38n64cwv8b6mdh9w3d7_zq/T/TemporaryItems/tmpaddon
1498745394620	addons.productaddons	INFO	downloadXHR File download. status=200
1498745394622	addons.productaddons	INFO	Downloaded file will be saved to /private/var/folders/0v/9tqxsqpn38n64cwv8b6mdh9w3d7_zq/T/TemporaryItems/tmpaddon
Disconnected from the target VM, address: '127.0.0.1:53199', transport: 'socket'

**Comment by lordofthejars** [comment]
I have tried on MacOS and Firefox 54 and it worked. Look can you clone this repo https://github.com/lordofthejars/examples and run this testhttps://github.com/lordofthejars/examples/blob/master/drone-graphene/drone-grahene-simple/src/test/java/arquillian/drone/graphene/simple/GoogleUITest.java

Test is failing but because of wrong assertion but Firefox was opened.

**Comment by punitbisht** [comment]
@lordofthejars You are right , example projects works fine with FF54. I am wondering whats wrong with my project ? Can you see the stack trace to identify problem ?

**Comment by punitbisht** [comment]
I verified the problem and the issue is dimension declaration in arquillian.xml file 
##### arquillian.xml file 
```
<arquillian xmlns="http://jboss.org/schema/arquillian"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="
        http://jboss.org/schema/arquillian
        http://jboss.org/schema/arquillian/arquillian_1_0.xsd">

    <extension qualifier="webdriver">
        <property name="browser">firefox</property>
        <property name="dimensions">1360x768</property>
    </extension>

</arquillian>

```

@lordofthejars if you add dimension property in your arquillian.xml file , you will get the same problem as I am .

**Comment by kvalev** [comment]
For some reason Drone sets the size twice, which causes Firefox to hang. The issue has been fixed in Firefox 55 Nightly. See the following GitHub issue: https://github.com/mozilla/geckodriver/issues/643

**Comment by punitbisht** [comment]
@kvalev Thanks for the info , lemme try this and I will update you with my findings .

**Comment by punitbisht** [comment]
Working fine with Firefox Nightly.

**Comment by MatousJobanek** [comment]
@kvalev big thanks for finding the root cause of the issue. And also thanks for pointing out that Drone sets the size twice - will be fixed in next release https://github.com/arquillian/arquillian-extension-drone/pull/103

**Comment by MatousJobanek** [comment]
Released as part of Drone 2.3.0
